api/src/pcapi/admin/custom_views/beneficiary_user_view.py

```diff
@@ -82,6 +83,34 @@ def beneficiary_deposit_type_formatter(view, context, model, name) -> Markup:
     )
 
 
+def beneficiary_total_amount_formatter(view, context, model, name) -> Markup:
+    amount = get_domains_credit(model)
+    all_spent = "0"
+    if amount is not None and amount.all is not None:
+        all_spent = amount.all.remaining
```

Comment:
(détail / style) Je ne suis pas un fan absolu du if ternaire, mais je trouve parfois que ça rend le code plus concis sans nuire à la lisibilité. Ici je pense que c'est le cas : 

```suggestion
    all_spent = amount.all.remaining if amount and amount.all else 0
```

Mais c'est peut-être une question de goût. :)

(idem dans les autres fonctions)